packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1234,7 +1310,7 @@ export class MergeTree {
 			const deltaSegments: IMergeTreeSegmentDelta[] = [];
 			const overlappingRemoves: boolean[] = [];
 			pendingSegmentGroup.segments.map((pendingSegment: ISegmentLeaf) => {
-				const overlappingRemove = !pendingSegment.ack(pendingSegmentGroup, opArgs);
+				const overlappingRemove = !ackSegment(pendingSegment, pendingSegmentGroup, opArgs);
```

Comment:
this shouldn't be done until ack is removed off segment, otherwise we have two copies of the same code to maintain, which are taking active development